keps/sig-node/1948-deallocate-dp-api/kep.yaml

```diff
@@ -0,0 +1,27 @@
+title: Add Deallocate and PostStopContainer to device plugin API
+kep-number: 1948
+authors:
+  - "@zvonkok"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: 2020-08-18
+
+reviewers:
+  - TBD
+approvers:
+  - TBD
+prr-approvers:
+  - "@deads2k"
```

Comment:
This looks like the "old" style `keyp.yaml` template. 
In the new style this info is captured in a separate file (in your case) called:
```
keps/prod-readiness/sig-node/1948.yaml
```
Please see https://github.com/kubernetes/enhancements/pull/2904/files for an example

Reply:
Thanks, updated.